internal/jobs/twitter.go

```diff
@@ -207,22 +207,33 @@ func (ts *TwitterScraper) ScrapeFollowersForProfile(j types.Job, baseDir string,
 }
 
 func (ts *TwitterScraper) ScrapeTweetsProfile(j types.Job, baseDir string, username string) (twitterscraper.Profile, error) {
-	scraper, account, _, err := ts.getAuthenticatedScraper(j, baseDir, TwitterScraperType)
+	logrus.Infof("[ScrapeTweetsProfile] Starting profile scraping for username: %s", username)
+	scraper, account, _, err := ts.getAuthenticatedScraper(j, baseDir, string(teetypes.TwitterJob))
 	if err != nil {
+		logrus.Errorf("[ScrapeTweetsProfile] Failed to get authenticated scraper: %v", err)
 		return twitterscraper.Profile{}, err
 	}
 	if scraper == nil {
+		logrus.Errorf("[ScrapeTweetsProfile] Scraper is nil after authentication")
 		return twitterscraper.Profile{}, fmt.Errorf("scraper not initialized for ScrapeTweetsProfile")
 	}
 
+	logrus.Infof("[ScrapeTweetsProfile] About to increment TwitterScrapes stat for WorkerID: %s", j.WorkerID)
```

Comment:
These messages should probably be `Debugf` so we can turn them off when everything is working fine and not get log spam.